dotcom-rendering/cloudformation.yml

```diff
@@ -181,6 +181,12 @@ Resources:
               - ssm:GetParameter
             Resource:
               - !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/frontend/*
```

Comment:
We should (perhaps separately from this PR) review and see if permissions for `/frontend/*` parameters are still needed in DCR

Reply:
The only place I can see SSM used is in [get-aws-ssm-parameters.js](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/scripts/dotenv/get-aws-ssm-parameters.js).

There is also [`apps-rendering`](https://github.com/guardian/dotcom-rendering/blob/main/apps-rendering/cdk/lib/mobile-apps-rendering.ts) references, but pretty certain there's no dependency here.